article/app/views/fragments/recipeArticleBody.scala.html

```diff
@@ -0,0 +1,175 @@
+@import layout.ContentWidths.MainMedia
+@import model.content.MediaAtom
+@import common.Localisation
+@(articleModel: ArticlePage, recipes: Seq[model.content.RecipeAtom])(implicit request: RequestHeader, context: _root_.model.ApplicationContext)
+
+@import common.LinkTo
+@import views.BodyCleaner
+
+@* Feb 2015: Google does only support NewsArticle in their amp carousel. To remove once Google starts supporting other schemas like schema.org/Review *@
```

Comment:
I don't think this is the case anymore. https://www.ampproject.org/docs/reference/spec